src/font/shaper/harfbuzz.zig

```diff
@@ -3,14 +3,17 @@ const assert = std.debug.assert;
 const Allocator = std.mem.Allocator;
 const harfbuzz = @import("harfbuzz");
 const font = @import("../main.zig");
+const terminal = @import("../../terminal/main.zig");
```

Comment:
... *sigh* this is a copy/paste mistake. It can be removed. (I don't have time right now)

Reply:
Wait no it isn't I just reordered it from below. I'm too tired for this 🙃